packages/rest/src/validation/request-body.validator.ts

```diff
@@ -87,10 +87,17 @@ function convertToJsonSchema(openapiSchema: SchemaObject) {
  * @param schema The JSON schema used to perform the validation.
  * @param globalSchemas Schema references.
  */
+export interface AnyObject {
+  // tslint:disable-next-line:no-any
+  [property: string]: any;
+}
+
+const compiledSchemaCache = new WeakMap();
+
 function validateValueAgainstJsonSchema(
   // tslint:disable-next-line:no-any
   body: any,
-  jsonSchema: object,
+  jsonSchema: SchemaObject,
```

Comment:
Please note that `SchemaObject` is referring to OpenAPI Spec schema, while this method is accepting JSON Schema. These two schemas are slightly different. Please revert this line.